client/init.c

```diff
@@ -300,6 +364,12 @@ int setup_miner(struct startup_parameters *parameters)
 		parameters->pool_address = pool_address_buf;
 	}
 
+    if(parameters->transport_flags & XDAG_DAEMON) {
```

Comment:
What is the purpose of creating daemon process for miner? This will cause issue with current GUI miners.

Reply:
miner in develop branch -d arg can not run daemon process , so i will add isGui flag to 
compatible with GUI miners